4/lib/model.d.ts

```diff
@@ -2518,9 +2518,9 @@ export interface Instance {
    * Emits null if and only if validation successful; otherwise an Error instance containing
    * { field name : [error msgs] } entries.
    *
-   * @param options.skip An array of strings. All properties that are in this array will not be validated
+   * @param options Options that are passed to the validator
    */
-  validate(options?: { skip?: string[] }): Promise<void>;
+  validate(options?: { skip?: string[], fields?: string[] }): Promise<ValidationError>;
```

Comment:
The return type is wrong.

> The promise fulfills if and only if validation successful; otherwise it rejects an Error instance containing { field name : [error msgs] } entries.